cmd/nodepool/openstack/create.go

```diff
@@ -3,8 +3,10 @@ package openstack
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	hyperv1 "github.com/openshift/hypershift/api/hypershift/v1beta1"
+	"github.com/openshift/hypershift/cmd/cluster/kubevirt/params"
```

Comment:
This suggests we should talk to the kubevirt folks about moving this package into a non-kubevirt library.

cc @davidvossel